### PR TITLE
Wangjun referencecommon

### PR DESCRIPTION
**Pull request by wangjun-microsoft**
bug fix: add the dependency "Microsoft.ProjectOxford.Common" to the nuget package.

### CODE REVIEW

File: ClientLibrary/FaceClientSDK.nuspec [hunk 1]
```diff
@@ -17,6 +17,7 @@
     <tags>Face Oxford ProjectOxford Microsoft Cognitive CognitiveServices</tags>
     <dependencies>
       <dependency id="Newtonsoft.Json" version="9.0.1" />
+      <dependency id="Microsoft.ProjectOxford.Common" version="[1.0.324)" />
```
Comment (huxuan) [inline ClientLibrary/FaceClientSDK.nuspec]: Please double check whether this kind of usage really works.

File: Sample-WPF/FaceAPI-WPF-Samples.csproj [hunk 2]
```diff
@@ -57,7 +57,7 @@
       <Private>True</Private>
     </Reference>
     <Reference Include="Microsoft.ProjectOxford.Face, Version=1.2.5.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```
Comment (huxuan) [inline Sample-WPF/FaceAPI-WPF-Samples.csproj]: I think there are many other places you need to change the version number. Please make sure all covered.

Comment (huxuan) [inline Sample-WPF/FaceAPI-WPF-Samples.csproj]: Besides the line here, other files that specify the version number:

- ClientLibrary/Properties/AssemblyInfo.cs
- Sample-WPF/Properties/AssemblyInfo.cs

Let me know if there is anything wrong I made.

**Comment by wangjun-microsoft** [comment]
Test is done already.

发自我的 iPhone

在 2017年4月20日，20:44，Xuan Hu (Sean) <notifications@github.com<mailto:notifications@github.com>> 写道：


@huxuan requested changes on this pull request.

________________________________

In ClientLibrary/FaceClientSDK.nuspec<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoft%2FCognitive-Face-Windows%2Fpull%2F26%23discussion_r112441914&data=02%7C01%7Cwangjun%40064d.mgd.microsoft.com%7C586f4e45948c488ab44e08d487eaec08%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636282890429616442&sdata=1qXDFY6Ml3CqIBnBmaVemuiZrkxSXBS2kyrKMufJ%2BG8%3D&reserved=0>:

> @@ -17,6 +17,7 @@
     <tags>Face Oxford ProjectOxford Microsoft Cognitive CognitiveServices</tags>
     <dependencies>
       <dependency id="Newtonsoft.Json" version="9.0.1" />
+      <dependency id="Microsoft.ProjectOxford.Common" version="[1.0.324)" />


Please double check whether this kind of usage really works.

________________________________

In Sample-WPF/FaceAPI-WPF-Samples.csproj<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoft%2FCognitive-Face-Windows%2Fpull%2F26%23discussion_r112442149&data=02%7C01%7Cwangjun%40064d.mgd.microsoft.com%7C586f4e45948c488ab44e08d487eaec08%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636282890429616442&sdata=IkGwasEfOtjeNLJODIZepW9NT1sExjz3PCcCh50v4aE%3D&reserved=0>:

> @@ -57,7 +57,7 @@
       <Private>True</Private>
     </Reference>
     <Reference Include="Microsoft.ProjectOxford.Face, Version=1.2.5.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">


I think there are many other places you need to change the version number. Please make sure all covered.

________________________________

In Sample-WPF/FaceAPI-WPF-Samples.csproj<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMicrosoft%2FCognitive-Face-Windows%2Fpull%2F26%23discussion_r112442638&data=02%7C01%7Cwangjun%40064d.mgd.microsoft.com%7C586f4e45948c488ab44e08d487eaec08%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636282890429616442&sdata=gFSrvKklLGybhCoJI3uZDusD0ziM176118z6kLGGPfY%3D&reserved=0>:

> @@ -57,7 +57,7 @@
       <Private>True</Private>
     </Reference>
     <Reference Include="Microsoft.ProjectOxford.Face, Version=1.2.5.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">


Besides the line here, other files that specify the version number:

  *   ClientLibrary/Properties/AssemblyInfo.cs
  *   Sample-WPF/Properties/AssemblyInfo.cs

Let me know if there is anything wrong I made.

File: ClientLibrary/FaceClientSDK.nuspec (hunk 1 above)
Comment (wangjun-microsoft) [inline ClientLibrary/FaceClientSDK.nuspec]: Fixed, thanks.

Comment (wangjun-microsoft) [inline ClientLibrary/FaceClientSDK.nuspec]: Fixed, thanks.

**Comment by wangjun-microsoft** [comment]
Leave a comment

**Pull request closed without merging**